frontend/src/lib/components/navbar/phone_navbar.svelte

```diff
@@ -42,10 +42,9 @@
     </div>
     <ThemeSelector />
     <div class="dropdown dropdown-end" on:focusout={handleDropdownFocusLost}>
-      <label
-        tabindex="0"
+      <div
         class="btn btn-ghost btn-sm rounded-btn text-xl"
-        on:click={handleDropdownClick}
+        on:keypress={handleDropdownClick}
```

Comment:
Pretty sure this removes clickability :( 
I think you need to define both on:click and on:keypress

Reply:
Fixed